pkg/testutils/echo/client/client.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"net"
+)
+
+func main() {
+	target := flag.String("target", "", "the server address")
+	payload := flag.String("message", "", "the message to send to the server")
+	flag.Parse()
+
+	if *target == "" || *payload == "" {
+		flag.Usage()
+		panic("invalid arguments")
+	}
+	conn, err := net.Dial("tcp", *target)
+	if err != nil {
+		panic(fmt.Sprintf("Failed to open connection to [%s] %v", *target, err))
+	}
+	defer conn.Close()
+
+	_, err = conn.Write([]byte(*payload))
+	if err != nil {
+		panic("Failed to send payload")
+	}
+	_, err = conn.Write([]byte("\n"))
```

Comment:
If we're adding a newline here why do we also add one at https://github.com/containernetworking/plugins/pull/526/files#diff-8466c05fa47a47331ae1d46458c3d11dR229 and https://github.com/containernetworking/plugins/pull/526/files#diff-8466c05fa47a47331ae1d46458c3d11dR229 ?

Reply:
Good catch! That's a leftover. Removed it.